external/fv3fit/fv3fit/_shared/predictor.py

```diff
@@ -75,53 +78,4 @@ def predict_columnwise(
         Returns:
             the predictions defined on the same dimensions as X
         """
-        if len(sample_dims) == 0 and feature_dim is None:
```

Comment:
You could consider emitting a [deprecation warning](https://docs.python.org/3/library/warnings.html) here.